src/app.js

```diff
@@ -36,10 +36,26 @@ async function getRedisStore () {
 // middleware
 app.use(helmet())
 
+const imgSrc = [
+  "'self'"
+]
+
+if (AppConstants.FXA_ENABLED) {
+  const fxaSrc = new URL(AppConstants.OAUTH_PROFILE_URI).origin
+  imgSrc.push(fxaSrc)
+
+  app.use(
+    helmet.crossOriginEmbedderPolicy({
+      policy: 'credentialless'
```

Comment:
I spent some time trying to figure this out.  `credentialless` is "experiemental" and [not supported by Firefox or Safari](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy).  Locally it _did_ work for me in Firefox, but Safari sure enough did not display the avatar.

I was wondering why the avatar works in the legacy site, but not the new version.  I believe it's because the new version loads ALL Helmet modules including crossOriginEmbedderPolicy which defaults to `require-corp`, which you overwrite to `credentialless`.  In the legacy site, we only opt-in to `contentSecurityPolicy` and `referrerPolicy`.  `crossOriginEmbedderPolicy` is unset and the avatar works on all browsers.

I believe @groovecoder set up Helmet on the legacy site and I'm curious on his take of what Helmet modules we should include and what we might omit – hoping `crossOriginEmbedderPolicy` is one we can do without for now.  If not, we'll have to see how this avatar behaves on Safari in production with `crossOriginEmbedderPolicy` enabled – maybe it's a non-issue, but I'm pessimistic on this.  We could also raise the issue with FxA to see if others have the avatar working with `crossOriginEmbedderPolicy` set to the supported default `require-corp`.